tools/system_libs.py

```diff
@@ -665,6 +664,8 @@ def get_files(self):
     if shared.Settings.WASM_BACKEND:
       # With the wasm backend these are included in wasm_libc_rt instead
       blacklist += [os.path.basename(f) for f in get_wasm_libc_rt_files()]
+    else:
+      blacklist += ['rintf.c']
```

Comment:
What this change?

Reply:
Blacklisting `rintf` forces Emscripten to import it from JavaScript. Instead, we want to use `rintf.c` on wasm backend because calls `__builtin_rintf`.